unittests/tools/test_sysdig_reports_parser.py

```diff
@@ -44,10 +44,9 @@ def test_sysdig_parser_missing_cve_field_id_from_csv_file(self):
             for finding in findings:
                 for endpoint in finding.unsaved_endpoints:
                     endpoint.clean()
-            self.assertTrue(
-                "sysdig report contains errors:" in str(context.exception)
-            )
-            self.assertTrue("ECONNREFUSED" in str(context.exception))
+        self.assertEqual(
+            "Number of fields in row (22) does not match number of headers (21)", str(context.exception)
+        )
```

Comment:
Should this be an `assertIn`? Or did the exception change?

```suggestion
        self.assertIn(
            "Number of fields in row (22) does not match number of headers (21)", str(context.exception)
        )
```

Reply:
Both are valid. The question is, how strict are going to be?